napari/layers/image/image.py

```diff
@@ -448,3 +448,18 @@ def to_xml_list(self):
             'image', width=width, height=height, opacity=opacity, **props
         )
         return [xml]
+
+    def _add_zarr_data(self, gp):
+        """Add layer data to zarr group."""
+        gp.array(
+            'data',
+            self.data,
+            shape=self.data.shape,
+            chunks=(128,) * len(self.data.shape),
```

Comment:
imho this should definitely be settable... Also, we should use *some* compression scheme, right?